src/render-delaying-extensions.js

```diff
@@ -62,12 +64,13 @@ export function waitForExtensions(win) {
  * Detects which, if any, render-delaying extensions are included on the page.
  * @param {!Window} win
  * @return {!Array<string>}
+ * @private
  */
-export function includedExtensions(win) {
+function includedExtensions(win) {
   const doc = win.document;
   dev.assert(doc.body);
 
-  return EXTENSIONS.filter(extension => {
+  return Object.keys(EXTENSIONS).filter(extension => {
```

Comment:
@dvoytenko had an important comment:

We need to make sure we call
`resources.schedulePass()`
right after the body tag was manipulated, because those changes might affect resource rendering. Should happen in separate CL, but it becomes more important after this change.